### ISSUE TITLE
teamcoco.py preload information could not be extracted

### ISSUE THREAD

**Issue by atomi** [rg3/youtube-dl#5626]
looks like it's failing here 
https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/teamcoco.py#L84

**Comment by jaimeMF** [comment]
Post [the full output with the --verbose option](https://github.com/rg3/youtube-dl/blob/master/README.md#bugs).

**Comment by atomi** [comment]
```
youtube-dl --verbose http://teamcoco.com/video/full-episode-mon-5-4-jeremy-renner-tj-miller-and-comedian-mikal-cronin
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--verbose', u'http://teamcoco.com/video/full-episode-mon-5-4-jeremy-renner-tj-miller-and-comedian-mikal-cronin']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2015.05.04
[debug] Python version 2.7.8 - Windows-8-6.2.9200
[debug] exe versions: none
[debug] Proxy map: {}
[Teamcoco] full-episode-mon-5-4-jeremy-renner-tj-miller-and-comedian-mikal-cronin: Downloading webpage
ERROR: Preload information could not be extracted
Traceback (most recent call last):
  File "youtube_dl\YoutubeDL.pyo", line 650, in extract_info
  File "youtube_dl\extractor\common.pyo", line 273, in extract
  File "youtube_dl\extractor\teamcoco.pyo", line 84, in _real_extract
ExtractorError: Preload information could not be extracted
```

**Comment by yan12125** [comment]
Can you watch the video in browsers?

Anyway, teamcoco extractor is broken, and I've fixed it in ac6c358c2ab882427f74af47fe5df762dd348c20. Now it works for at least the three test cases in youtube-dl.

**Comment by atomi** [comment]
@yan12125 I'll close this issue and open a new issue if ac6c358 fails. Thanks.

**Comment by yan12125** [comment]
teamcoco changed again, and it's obvious that their work is directed against youtube-dl.

**Comment by yan12125** [comment]
I managed to fix it in 3a105f7b20e8a3f742ac86cc1a6b02935b831778. Currently teamcoco extractor works.
